### PR TITLE
namespaceWhitelist deprecation docs

### PR DESCRIPTION
**Pull request by f0ssel**

### CODE REVIEW

File: changelog/1.17.0.md [hunk 1]
```diff
@@ -5,13 +5,15 @@ description: "Released on 03/17/2021"
 
 ### Breaking Changes ❗
 
-- (?) Wsp orgs migration
+- The `namespaceWhitelist` field on the Coder helm charts has been removed and
```
Comment (vapurrmaid) [inline changelog/1.17.0.md:8]:
```suggestion
- helm: The `namespaceWhitelist` field on the Coder helm charts has been removed and
```

File: admin/access-control/organizations.md [hunk 2]
```diff
@@ -312,13 +312,23 @@ organization.
 
 ## Namespaces
 
-Coder's helm chart includes `namespaceWhitelist`. This value takes a list of the
-namespaces that are in your cluster and are available to Coder. If you don't set
-`namespaceWhitelist`, the only available namespace is the one Coder has deployed
-to (this is also the default namespace).
+NOTICE: The `namespaceWhitelist` field has been deprecated in Coder version 
+1.17.
```
Comment (vapurrmaid) [inline admin/access-control/organizations.md]: I think we can call it out like:

```md
> **Notice**: The `namespaceWhitelist` field as been deprecated in Coder version 1.17.
```

It should render as a nice callout

File: admin/access-control/organizations.md [hunk 3]
```diff
@@ -312,13 +312,23 @@ organization.
...
 
-If you're a site manager, you can assign a namespace to an organization; all
-resources for that organization now reside in that namespace. **Please note that
-this field is not editable once set.**
+Coder's helm chart previously included a `namespaceWhitelist` field that has 
+been removed in version 1.17. This value took a list of the namespaces that are 
+in your cluster and are available to Coder.
```
Comment (vapurrmaid) [inline admin/access-control/organizations.md]: If we add the notice to the top, I think we can keep the old wording and just use parentheses with a bold statement like

```md
- (**deprecated in 1.17**) If you're a site manager, you can assign...
```

Comment (f0ssel) [reply admin/access-control/organizations.md]: Could you give a full example of the format you are thinking? I'm having a hard time figuring out where I would add that back cleanly.

Comment (f0ssel) [reply admin/access-control/organizations.md]: I've pushed a change with the notice header

Comment (vapurrmaid) [reply admin/access-control/organizations.md]: It looks good - all good

**Review by vapurrmaid — APPROVED** [review]

**Review by khorne3 — APPROVED** [review]

**Comment by mterhar** [comment]
do you want to give me an hour to walk through the provisioning process to give code snippets?

**Comment by f0ssel** [comment]
I have that in the linked docs "Deploying a Workspace Provider"

**Comment by f0ssel** [comment]
https://github.com/cdr/docs/pull/166/files#diff-7ff549a0e37eb25c201b1fb6d7c634f291d1d55785a52b404b730ab45ecc1fd4R1

**Comment by f0ssel** [comment]
I'll go ahead and merge this into the release branch, and feel free to open a new PR if you wanna add / edit anything further

**Comment by mterhar** [comment]
Sounds good. I want to pave the way from whitelist to providers a bit better.

**Pull request merged**